test/extended/conformance-k8s.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# Runs the Kubernetes conformance suite against an OpenShift cluster
+#
+# Test prerequisites:
+#
+# * all nodes that users can run workloads under marked as schedulable
+#
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+
+# Check inputs
+if [[ -z "${KUBECONFIG-}" ]]; then
+  os::log::fatal "KUBECONFIG must be set to a root account"
+fi
+test_report_dir="${ARTIFACT_DIR}"
+mkdir -p "${test_report_dir}"
+
+kubernetes="${KUBERNETES_ROOT:-${OS_ROOT}/../../../k8s.io/kubernetes}"
+if [[ ! -d "${kubernetes}" ]]; then
+  os::log::fatal "Cannot find Kubernetes source directory, set KUBERNETES_ROOT"
+fi
+version="${KUBERNETES_VERSION:-v1.7.6}"
+oc="$( os::util::find::built_binary oc )"
```

Comment:
Unless you're worried about using a system `oc` when a built one exists, this isn't necessary. Expect that it exists and just use `oc` directly

Reply:
Hrm, yeah, I guess so.